### PR TITLE
Change Password By Email

### PR DESCRIPTION
**Pull request by mhdip**

### CODE REVIEW

File: NMS API-N/Controllers/AuthController.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@
 using NMS_API_N.Extension;
 using NMS_API_N.IServices;
 
+
```
Comment (D-And-S) [inline NMS API-N/Controllers/AuthController.cs:11]: Why to line space here

File: NMS API-N/Controllers/AuthController.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@
...
+
```
Comment (D-And-S) [inline NMS API-N/Controllers/AuthController.cs:12]: extra line

File: NMS API-N/Controllers/AuthController.cs [hunk 3]
```diff
@@ -51,6 +53,7 @@ public async Task<ActionResult<UserDto>> Register(RegisterDto registerDto)
                 Token = await _tokenService.CreateToken(user),
                 EmployeeId = user.EmployeeId,
             };
+
```
Comment (D-And-S) [inline NMS API-N/Controllers/AuthController.cs:56]: Extra Line

File: NMS API-N/Controllers/AuthController.cs [hunk 4]
```diff
@@ -72,29 +75,6 @@ public async Task<ActionResult<UserDto>> Login(LoginDto loginDto)
             };
         }
```
Comment (D-And-S) [inline NMS API-N/Controllers/AuthController.cs:77]: What did you do to this method

File: NMS API-N/Model/Entities/User.cs [hunk 5]
```diff
@@ -17,7 +17,8 @@ public class User : IdentityUser<int>
         public bool? IsActive { get; set; }
         public bool? IsDelete { get; set; }
         public DateTime? DeletedDate { get; set; }
-
```
Comment (D-And-S) [inline NMS API-N/Model/Entities/User.cs:20]: remove this extraline

File: NMS API-N/Unit Of Work/UnitOfWork.cs [hunk 6]
```diff
@@ -23,6 +23,7 @@ public UnitOfWork(DataContext context, IMapper mapper)
         public ICountryRepository CountryRepository => new CountryRepository(_context, _mapper);
```
Comment (D-And-S) [inline NMS API-N/Unit Of Work/UnitOfWork.cs:24]: Remove extra line

**Review by D-And-S — COMMENTED** [review]
You have put lot of extra line and please explain why did remove those function ?

File: NMS API-N/Controllers/AuthController.cs (hunk 4 above)
Comment (mhdip) [reply NMS API-N/Controllers/AuthController.cs:77]: Well, initially i defined this method in AuthController but it belongs to SettingsController. that's why i switched this method to SettingsController

**Pull request closed without merging**